dashboard/models.py

```diff
@@ -1481,12 +1661,14 @@ class RedcapRecord(db.Model):
     date = db.Column('entry_date', db.Date)
     user = db.Column('redcap_user', db.Integer)
     comment = db.Column('comment', db.Text)
-    redcap_version = db.Column('redcap_version', db.String(10), default='7.4.2')
+    redcap_version = db.Column('redcap_version',
+                               db.String(10),
+                               default='7.4.2')
```

Comment:
This doesn't need to be addressed now but is there a way to get the redcap version from the DET?  Just so we can stop hardcoding this value.

Reply:
I'm actually not sure! Let me play with that today. I think the hardcoding might have just been a tiny bit of a lazyness when Tom first added that feature that got brought forward with every update.

Reply:
Ah! Right. This particular hard coded instance is just so that we can safely have a 'not null' on the version column. dashboard/redcap.py does parse the DET for the version of the actual server. So the default version number should never be used, really.

Reply:
Cool! Thank you for checking!